### ISSUE TITLE
LoadError: JuliaTeam.toml": No such file or directory julia

### ISSUE THREAD

**Issue by logankilpatrick** [JuliaComputing/JuliaProIssues#1]
https://stackoverflow.com/questions/63962447/julia-loaderror-juliateam-toml-no-such-file-or-directory

**Comment by bmharsha** [comment]
Inorder to fix this issue, end-user needs to update their version of `Blosc` to v0.7.0 , this is the latest version as of now and installing HDF5 on a vanilla JuliaPro v1.5.1-1 installation fetches this version by default.

```
julia> Pkg.add("HDF5")
   Updating registry at `~/.julia/registries/General`
######################################################################## 100.0%
   Updating registry at `~/.julia/registries/JuliaComputingRegistry`
######################################################################## 100.0%
  Resolving package versions...
  Installed Blosc_jll ─ v1.14.3+1
  Installed HDF5_jll ── v1.10.5+6
  Installed Lz4_jll ─── v1.9.2+2
  Installed Blosc ───── v0.7.0
  Installed HDF5 ────── v0.13.6
Downloading artifact: Blosc
Downloading artifact: HDF5
Downloading artifact: Lz4
Updating `~/.julia/environments/JuliaPro_v1.5.1-1/Project.toml`
  [f67ccb44] + HDF5 v0.13.6
Updating `~/.julia/environments/JuliaPro_v1.5.1-1/Manifest.toml`
  [a74b3585] + Blosc v0.7.0
  [0b7ba130] + Blosc_jll v1.14.3+1
  [f67ccb44] + HDF5 v0.13.6
  [0234f1f7] + HDF5_jll v1.10.5+6
  [5ced341a] + Lz4_jll v1.9.2+2
   Building HDF5 → `~/.julia/packages/HDF5/T1b9x/deps/build.log`

```

**Comment by bmharsha** [comment]
We have now removed our patched BinaryProvider from JuliaComputingRegistry, this should fix the issue permanently for all JuliaPro v1.5.x users because they will be using opensource BinaryProvider from now on.
